lib/beaker/host/unix/pkg.rb

```diff
@@ -87,10 +87,10 @@ def install_package(name, cmdline_args = '', version = nil, opts = {})
       execute("zypper --non-interactive --gpg-auto-import-keys in #{name}", opts)
     when /el-4/
       @logger.debug("Package installation not supported on rhel4")
-    when /amazon-2023|fedora-(2[2-9]|3[0-9])/
+    when /amazon-2023|el-(8|9|1[0-9])|fedora-(2[2-9]|3[0-9])/
```

Comment:
Fedora 40 is about to come out. Any thoughts on assuming DNF?
```ruby
    when /cisco|eos|el-[1-7]-/
      # yum
    when /amazon|el|fedora/
      # dnf
```

Reply:
Sounds good to me. I guess at this point,we can assume any Fedora < 22 machines running are long gone.

I'll add another commit